client/src/pages/private/ParticipantWithdrawConfirm.js

```diff
@@ -36,7 +36,7 @@ export default () => {
           variant='contained'
           onClick={() => history.push(Routes.ParticipantLanding)}
         >
-          Return Home
```

Comment:
I think we want to leave this as-is, this page only returns to the homepage whereas the other one returns to the PEOI